templates/myaccount/related-orders.php

```diff
@@ -99,4 +103,31 @@
 	</tbody>
 </table>
 
+<div class="woocommerce-subscriptions-related-orders-pagination-links woocommerce-pagination">
+	<?php
+	$prev_page = max( 1, $page - 1 );
+	$next_page = min( $max_num_pages, $page + 1 );
+
+	$page_link = static function( int $page, string $type, string $label, bool $enabled = true ): string {
+		$base_classes = 'button woocommerce-button wp-element-button';
+		$classes      = esc_attr( $base_classes . ' woocommerce-button--' . $type . ( $enabled ? '' : ' disabled' ) );
+		$url          = esc_url( add_query_arg( 'related_orders_page', $page, '#woocommerce-subscriptions-related-orders-table' ) );
+
+		return "<a href='$url' class='$classes'>$label</a>";
+	};
+
+    // phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped -- Our helper function takes care of escaping.
+	?>
+	<div class="pagination-links">
+		<?php echo $page_link( 1, 'first', _x( 'First', 'Related orders pagination', 'woocommerce-subscriptions' ), 1 !== $page ); ?>
+		<?php echo $page_link( $prev_page, 'prev', _x( 'Previous', 'Related orders pagination', 'woocommerce-subscriptions' ), 1 !== $page ); ?>
+	</div>
+
+	<div class="pagination-links">
+		<?php echo $page_link( $next_page, 'next', _x( 'Next', 'Related orders pagination', 'woocommerce-subscriptions' ), $page < $max_num_pages ); ?>
+		<?php echo $page_link( $max_num_pages, 'last', _x( 'Last', 'Related orders pagination', 'woocommerce-subscriptions' ), $page < $max_num_pages ); ?>
+	</div>
+	<?php // phpcs:enable ?>
+</div>
```

Comment:
I like this implementation a lot, well done! I've made another round of testing, and noticed that layout looks terrible on mobile devices: 

![CleanShot 2025-04-28 at 11 24 30@2x](https://github.com/user-attachments/assets/d14e07d6-2461-4e90-aa3b-7b269ab537ae)

Let's update markup and styling for that:
- Replace text on buttons with "<<" for first and "<" for pervious.
- Fix buttons alignment on the right edge.